mlir/include/mlir/Transforms/DialectConversion.h

```diff
@@ -92,9 +95,9 @@ class TypeConverter {
     /// used if the new types are not intended to remap an existing input.
     void addInputs(ArrayRef<Type> types);
 
-    /// Remap an input of the original signature to another `replacement`
+    /// Remap an input of the original signature to another `replacements`
     /// value. This drops the original argument.
-    void remapInput(unsigned origInputNo, Value replacement);
+    void remapInput(unsigned origInputNo, ArrayRef<Value> replacements);
```

Comment:
```suggestion
    void remapInput(unsigned origInputNo, ValueRange replacements);
```
Not sure this makes a large difference for most callers, but I would prefer `ValueRange` over `ArrayRef` generally speaking

Reply:
This was on purpose because the other overloads in this class take an `ArrayRef<Type>`.